stemcell_builder/stages/system_google_packages/apply.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# -*- encoding: utf-8 -*-
+# Copyright (c) 2014 Pivotal Software, Inc. All Rights Reserved.
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+# Copy google daemon packages into chroot
+cp -R $assets_dir/usr $chroot/
+
+if [ -f $chroot/etc/debian_version ] # Ubuntu
```

Comment:
perhaps we can use [`get_os_type`](https://github.com/cloudfoundry/bosh/blob/d2f73ee7636f2325bf6998670228682d194627c9/stemcell_builder/lib/prelude_apply.bash#L20) instead of duplicating logic